blocks/columns/columns.css

```diff
@@ -176,7 +176,23 @@ main .section-bottom .columns-2-cols > div >:nth-child(2) p:nth-of-type(3)  {
         font-size: 128px;
         }
 
-    main .section-top-grid >div:nth-of-type(2) > div:nth-of-type(2) {
+    .section-top-grid > div:nth-child(1) > div:nth-child(1), p, .article-content-wrapper ul {
+        font-family: var(--sans-serif-font-light);
+        font-size: var(--font-size-21);
+        font-weight: var(--font-weight-light);
+        line-height: var(--line-height-160);
+        letter-spacing: var(--letter-spacing-1);
+        }
+
+    main .section-top-grid > div:nth-of-type(2) > div:nth-of-type(2) > h1, main .section-top-grid > div:nth-of-type(2) > div:nth-of-type(2) > p {
```

Comment:
Can you add a new line after `,`? When I first saw this I thought it was one giant selector. 😮 There is a lot of trust in the HTML markup here.

Reply:
Agreed on your 'trust' concern. I saw a lot of this, but I'm not close enough to project to know how best to change it (at least not yet).

Reply:
Makes sense. No sense rocking the boat too much.